lib/annotate/annotate_models.rb

```diff
@@ -913,6 +908,7 @@ def map_col_type_to_ruby_classes(col_type)
       when 'date'                                          then Date.to_s
       when 'text', 'string', 'binary', 'inet', 'uuid'      then String.to_s
       when 'json', 'jsonb'                                 then Hash.to_s
+      when 'boolean'                                       then 'Boolean'
```

Comment:
Does this change any current behavior?

Reply:
Seems not. Boolean is a class that YARD understand quite well. It's better than TrueClass and FalseClass

Reply:
If you are referring to just the boolean case, `Boolean` is the string for what YARD expects a boolean type to be in the comments.

Reply:
Gotcha